docs/plugins/development/stable-plugin-descriptor.properties

```diff
@@ -0,0 +1,57 @@
+# Elasticsearch plugin descriptor file
+# This file must exist as 'plugin-descriptor.properties' inside a plugin.
+#
+### example plugin for "foo"
+#
+# foo.zip <-- zip file for the plugin, with this structure:
+# |____   <arbitrary name1>.jar <-- classes, resources, dependencies
+# |____   <arbitrary nameN>.jar <-- any number of jars
+# |____   plugin-descriptor.properties <-- example contents below:
+#
+# classname=foo.bar.BazPlugin
+# description=My cool plugin
+# version=6.0
+# elasticsearch.version=6.0
+# java.version=1.8
+#
+### mandatory elements for all plugins:
+#
+# 'description': simple summary of the plugin
+description=${description}
+#
+# 'version': plugin's version
+version=${version}
+#
+# 'name': the plugin name
+name=${name}
+#
+# 'java.version': version of java the code is built against
+# use the system property java.specification.version
+# version string must be a sequence of nonnegative decimal integers
+# separated by "."'s and may have leading zeros
+java.version=${javaVersion}
+#
+# 'elasticsearch.version': version of elasticsearch compiled against
+elasticsearch.version=${elasticsearchVersion}
```

Comment:
would it make sense to mention that with stable plugins this version only have to match a major from the elasticsearch cluster.
so we can have a elasticsearch.version=8.0.0 and it would be fine to run on 8.5.0 ES cluster.
when previously it would have to match exactly
wdyt @williamrandolph ?

Reply:
This file is actually not being used. The file at /build-tools/src/main/resources is included instead. I think it would be good to update that file - also on line 2 w.r. to the name.

Reply:
good point - will update this too

Reply:
https://github.com/elastic/elasticsearch/pull/93501